base_external_dbsource_mysql/models/base_external_dbsource.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2011 Daniel Reis
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import logging
+
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from odoo.addons.base_external_dbsource.models import base_external_dbsource
+
+    CONNECTORS = base_external_dbsource.BaseExternalDbsource.CONNECTORS
+    try:
+        import MySQLdb
+
+        CONNECTORS.append(("mysql", "MySQL"))
```

Comment:
Why not handle this exactly like the sqlalchemy import, and do away with the assert?